Tasks/InstallSSHKey/task.json

```diff
@@ -42,6 +42,14 @@
             "helpMarkDown": "The contents of the public SSH key."
         },
         {
+            "name": "sshPassphrase",
+            "type": "string",
+            "label": "SSH Passphrase",
+            "defaultValue": "",
+            "required": false,
+            "helpMarkDown": "The passphrase for the SSH key."
```

Comment:
We could further indicate that a passphrase is optional with the following:
The passphrase for the SSH key, **if any**.